### DISCUSSION TITLE
Announcing:  Project Board swimlanes & Create issues in repository groups 🏊

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#61871] [Projects and Issues]
Today's Issues & Projects update brings you board swimlanes and the ability to create issues in repository groups!

## 🏊 Board swimlanes
You can now configure swimlanes on your boards by selecting a Group by field from the view configuration menu. This allows you to break up your items by different workstreams, team members, or priorities, similar to groups on tables and roadmaps. Drag and drop your items between columns and groups to quickly make adjustments, or add a new item directly.

![project board swimlanes](https://github.com/community/community/assets/92824518/b9fc1fb7-7d6d-4f89-82c1-4c1702516cd4)

Here is a video showing how these are enabled.
https://github.com/community/community/assets/101840513/ed62861c-8f31-49d3-9bce-7fe335afb8e3

## ➕ Create issues in repository groups
You can now create issues when grouped by Repository on the table and roadmap layout. Click Create new issue or start typing the title to get started.

![repository groups](https://github.com/community/community/assets/92824518/be086646-5feb-45f8-818b-e9047c79753c)

See how to use GitHub for project planning with [GitHub Issues](http://github.com/features/issues), check out what's on the [roadmap](https://github.com/orgs/github/projects/4247/views/7), and learn more in the [docs](https://docs.github.com/issues).

**Comment by cilerler** [comment]
Appreciate the new feature. Noticed `Estimate: *` in the screenshots - is this an upcoming feature or has it already been released?

**Reply by rileybroughten** [reply]
@cilerler this is a custom field labeled "Estimate" that is then shown on the group headers as a field sum to calculate the sum of that field for that group of items. Here is some more information in the [documentation](https://docs.github.com/en/issues/planning-and-tracking-with-projects/customizing-views-in-your-project/customizing-the-board-layout#showing-the-sum-of-a-number-field)!

<img width="749" alt="Screenshot 2023-07-31 at 7 45 53 AM" src="https://github.com/community/community/assets/101840513/0c69029e-18fb-42b6-83b2-80a5ab91ebeb">

**Reply by cilerler** [reply]
Much appreciated! I overlooked that feature somehow. 🤗

**Comment by hamzahc1** [comment]
@rileybroughten How are the groups ordered? We use milestones for the grouping but it doesn't appear to be alphabetic? Is there a way to change the order the groups appear? 
![CleanShot 2023-08-02 at 09 31 35](https://github.com/community/community/assets/14341285/6c0a88ca-26b8-4618-ab59-73f63cd95969)

**Reply by rileybroughten** [reply]
Thank you for the feedback! Currently you cannot reorder how milestones appear if you are grouping by them, but we have noted your feedback on this 👍

**Reply by syradar** [reply]
How is it calculating the order? Is the emoji codepoint used if that is the first character in the milestone name or is it milestone number?

**Comment by tdrayna** [comment]
@rileybroughten Hi, I'm trying to use the task board for an Agile sprint and I'm not seeing the _Group:_ setting when I configure the view. I see it under the Table style tab and I can group by Assignee, but not under the Board style tab. But reading above this should be possible right?

We are using GHES 3.9.2 at the U of MN.

![image](https://github.com/community/community/assets/128559955/9ea04698-3e5a-4293-a714-efcce3d63bb3)

**Reply by rileybroughten** [reply]
Board swimlanes will be included in GHES 3.11, here is the[ public roadmap issue](https://github.com/orgs/github/projects/4247/views/7?pane=issue&itemId=30227648) where you can follow along for updates once this is released on GHES (see the `GHES 3.11` label) 👍

**Reply by tdrayna** [reply]
Thanks! I tested this out on GitHub.com and I definitely like the new sorting flexibility. Also the custom labels (E.g. estimate) that can be summed --perfect for sprints.

**Comment by mtcA6** [comment]
The title of the swimlane is left adjusted and shouldn't be. 
For specifics see screenshots here in [this comment](https://github.com/orgs/community/discussions/63398#discussioncomment-6740947)

**Reply by rileybroughten** [reply]
Group headers are now sticky as you scroll, which keeps the header of your swimlane in view as you scroll to other columns - see the changelog announcement [here](https://github.blog/changelog/2023-09-07-github-issues-projects-september-7th-update/#scroll-sticky-group-headers)! ✨ Let us know if you have any additional feedback on this!
